### PR TITLE
[NT-1676] Argo Removal Clean-up

### PR DESCRIPTION
**Pull request by ncreated**
# 📲 What

Hello 👋. Following the "Getting Started" lead me to having build issues (`Xcode 12.0 (12A7209)`). I fixed it on my end by doing following, which you may consider integrating on your end:
* listing `thoughtbot/Runes` dependency in `Cartfile`,
* cleaning up references to `Argo` dependency removed in #1327

# 🤔 Why

`thoughtbot/Runes` is a required dependency, but `make bootstrap` didn't lead to installing it because it was not listed in `Cartfile`. As `make bootstrap` calls `carthage update --platform iOS`, it should be there IMHO.

`Argo` framework was still referenced in many places, so it was breaking the build (e.g. when processing `xcfilelist`).

### CODE REVIEW

**Comment by justinswart** [comment]
hey @ncreated thanks for flagging. It does seem like there were a couple things missed from the removal of the Argo dependency in #1323. I'll push a couple more commits to this branch and we can get this merged 👍

File: Cartfile.resolved [hunk 1]
```diff
@@ -1,6 +1,6 @@
 binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAnalyticsBinary.json" "6.32.0"
 binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseCrashlyticsBinary.json" "6.32.0"
-github "Alamofire/Alamofire" "5.2.2"
+github "Alamofire/Alamofire" "5.4.0"
```
Comment (ncreated) [inline Cartfile.resolved:3]: @justinswart indeed, I've been considering pushing this, but I wasn't sure if this bump doesn't break anything 👌.

Comment (justinswart) [reply Cartfile.resolved:3]: I think this is fine, we only use AlamofireImage for some simple image caching - so this is included by way of that dependency.

**Review by singhhari — APPROVED** [review]
lgtm!

**Review by justinswart — APPROVED** [review]
Will merge once tests pass 👍

**Pull request merged**